src/GraphQL/Authorization/AuthorizationExtensions.cs

```diff
@@ -63,6 +80,76 @@ public static TMetadataProvider AuthorizeWith<TMetadataProvider>(this TMetadataP
             return provider;
         }
 
+        /// <inheritdoc cref="AuthorizeWithPolicy{TMetadataProvider}(TMetadataProvider, string)"/>
+        [Obsolete("Please use AuthorizeWithPolicy instead.")]
```

Comment:
```suggestion
        [Obsolete("Please use AuthorizeWithPolicy instead. Will be removed in v6.")]
```

I suggest to always write when obsoleted member will be removed.

Reply:
Agree

Reply:
Done